.github/templates/main.yml

```diff
@@ -367,23 +371,24 @@ jobs:
         shell: powershell
       - name: Run the tests
         run: |
-          echo "test_results=$env:LOCALAPPDATA/Packages/$(get-appxpackage -name realm.uwp.tests | select -expandproperty PackageFamilyName)/LocalState/TestResults.UWP.xml" | Out-File $env:GITHUB_ENV -Encoding utf8 -Append
+          echo "test_results=$env:LOCALAPPDATA\Packages\$(get-appxpackage -name realm.uwp.tests | select -expandproperty PackageFamilyName)\LocalState\TestResults.UWP.xml" | Out-File $env:GITHUB_ENV -Encoding utf8 -Append
```

Comment:
Backslashes in place of forwardslashes is the outcome of testing with the new test reporting GH action. It's simply done to have more consistently looking paths. I did not want the action to look at paths like `C:\bd\s\e/g/u/k`; since I set the option of `path-replace-backslashes: true`, I didn't want to risk to confuse the GH action on what to do in mixed path. And while I could have gone to look at the source code to see how the forwardslashes are replaced when that option is on, I don't see the point in having mixed path anyway.

TL;DR: Backslashes instead of forwardslashes is mainly done for consistent style given the fact that we're in powershell. This avoids potential issues with the new GH action for reporting test result.